### PR TITLE
ADEN-5649 | Make VUAP Oasis tests more stable

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-5649

@Wikia/adeng

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java [hunk 1]
```diff
@@ -217,19 +222,15 @@ private boolean isOverlayNoVisible() {
     return wait.forElementNotVisible(By.cssSelector(String.format(PAUSE_BUTTON_SELECTOR_FORMAT, slot)));
   }
 
-  public boolean isResolvedStateDisplayed(double defaultVideoHeight, double resolvedVideoHeight) {
-    return EXPECTED_PERCENTAGE_DIFFERENCE_IN_VIDEO_AD_HEIGHT == getStatesPercentageDifference(defaultVideoHeight, resolvedVideoHeight);
+  private boolean isPrefooterNoVisible() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java]: MAJOR Remove this unused private "isPrefooterNoVisible" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java [hunk 2]
```diff
@@ -186,7 +187,14 @@ public void vuapResolvedStateShouldHaveLinkToFandomArticleOnImage(Page page, Str
   public void vuapResolvedStateShouldCloseAfterTapingOnCloseButton(Page page, String slot, String videoIframeSelector) {
     final AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
     final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+    // make sure slot is rendered so we get resolved state after the refresh
+    scrollToSlot(slot, ads);
+    vuap.replay();
+
     ads.refreshPage();
+    // TODO remove wait in ADEN-5481
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java]: INFO Complete the task associated to this TODO comment. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 2 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L221](https://github.com/Wikia/selenium-tests/blob/1f9377742ab93ac750be31fe9727fa432f8cef2d/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L221): Remove this unused private "isOverlayNoVisible" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java [hunk 3]
```diff
@@ -217,19 +222,15 @@ private boolean isOverlayNoVisible() {
     return wait.forElementNotVisible(By.cssSelector(String.format(PAUSE_BUTTON_SELECTOR_FORMAT, slot)));
   }
 
-  public boolean isResolvedStateDisplayed(double defaultVideoHeight, double resolvedVideoHeight) {
-    return EXPECTED_PERCENTAGE_DIFFERENCE_IN_VIDEO_AD_HEIGHT == getStatesPercentageDifference(defaultVideoHeight, resolvedVideoHeight);
+  private boolean isPrefooterNotVisible() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java]: MAJOR Remove this unused private "isPrefooterNotVisible" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 2 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L221](https://github.com/Wikia/selenium-tests/blob/b21738415fbbe813837a89d94e7823c7191ae24e/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L221): Remove this unused private "isOverlayNoVisible" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java (hunk 3 above)
Comment (kvas-damian) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java]: it is used...

**Pull request merged**